NEWS.md

```diff
@@ -1,6 +1,21 @@
 erGW - 3GPP GGSN and PDN-GW in Erlang
 =====================================
 
+Version 2.5.0 - 31 July 2020
+----------------------------
+* implement path monitoring profiles for outgoing (active) paths
+  * different echo interval configurable depending whether path is 
+    active (has contexts), idle (no contexts) or is down
+  * Monitoring of max duration of Idle and Down paths
+* rewrite state logic of gtp path
+* make path management defaults configurable
+* bump ergw_aaa dependency to 3.5.0 to support state stats and 
+  diameter avp filters
+* bump prometheus_diameter_collector to report plain error counters
+* fix: start echo sending after getting a response or timeout
```

Comment:
this is part of that path monitoring and doesn't need a seperate point in the NEWS update